php/php.editor/src/org/netbeans/modules/php/editor/verification/UnusableTypesUnhandledError.java

```diff
@@ -229,6 +258,16 @@ private void checkReturnType(@NullAllowed Expression returnType, boolean isInUni
             } else if (type instanceof UnionType) {
                 ((UnionType) type).getTypes().forEach(unionType -> checkReturnType(unionType, true));
                 checkUnionType((UnionType) type);
+            } else if (type instanceof Identifier) {
+                // method, labmda funciton, and arrow function can use static return type
```

Comment:
Nitpick: Typos, "la[b]mda func[i]ton".

Reply:
Thanks for caching them :) Will fix them later. (Just add the `TODO` label)